synthesizer/synthesize.py

```diff
@@ -8,13 +8,15 @@
 import numpy as np
 from pathlib import Path
 from tqdm import tqdm
-
+#@tomcattwo added below line to fix Win10 pickle error per issue #669 and  blue-fish/Real-Time-Voice-Cloning@89a9964
```

Comment:
Please remove all new comments. This information will be summarized in the commit message.